frontend/templates/views/partials/version-toggle.j2

```diff
@@ -11,4 +11,14 @@
   </select>
 </div>
 {% endif %}
+
+<div>
+{% do doc.styles.addCssFile('css/components/atoms/pill.css') %}
+{% if doc.experimental %}
+  <div class="ap-a-pill ap-a-pill-small ap-a-pill-meta">Experimental</div>
+{% endif %}
+{% if doc.deprecated %}
+  <div class="ap-a-pill ap-a-pill-small ap-a-pill-meta">Deprecated</div>
+{% endif %}
+</div>
 </h1>
```

Comment:
Hm... having this partial now named `version-toggle` seems confusing. Can you update the partial name to `component-headline` for example?